.gitignore

```diff
@@ -1,5 +1,11 @@
-/aiogram_dialog.egg-info/
+aiogram_dialog.egg-info/
 __pycache__
 /build
 .idea/
 /dist
+
+.venv
```

Comment:
Why should we ignore these files?

Reply:
I can't understand why do you ask it. But you can ignore all my changes in '.gitignore'.
I tried to exclude it from commit, but I couldn't do it (I'm not a very good GIT user yet).

Reply:
Figured out how to create my local exclude list for git. Removed my changes from .gitignore